### ISSUE TITLE
Could you give some detailed information to configure new project based on ErizoClientOS

### ISSUE THREAD

**Issue by wilsonting8fang** [zevarito/ErizoClientIOS#2]
Hi Zevarito,

I am trying to create my project based on ErizoClientOS and following the process below, but I didn't know exactly how I should do.

1. Link result library into your project

2.  Add search path into your project

Could you please explain it more? 

Thanks,
Wilson

**Comment by zevarito** [comment]
@wilsonting8fang

After successfully build the library, I think you have two options at least:

1) with XCode

* Create a new XCode project.
* Drag `libErizoClient.a` into your project.
* `Build Phases` > `Link Binary With libraries` add libErizoClient.a
* `Build Settings` > `Search Paths` > `Header Search Paths` add an entry pointing to ErizoClientIOS sources, and make it recursive.

2) with Pods

You could start a new CocoaPods project with the following configuration:

```
platform :ios, :deployment_target => "8.4"

target “ErizoClient” do
  platform :ios, :deployment_target => "8.4"
  xcodeproj “ErizoClientIOS”
  pod "libjingle_peerconnection"
end
```

Let me know if it worked.

**Comment by wilsonting8fang** [comment]
@zevarito 

Thanks for your quick response.  Basically I followed the steps you showed me, it is much better. But after the second step(with Pods), I added the code "[ErizoClient sharedInstance];" and build the project. I got the following error.

Undefined symbols for architecture armv7:
  "_utf8_nextCharSafeBody", referenced from:
      _validate_dispatch_data_partial_string in libErizoClient.a(SRWebSocket.o)
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)
